src/proxymatic/discovery/marathon.py

```diff
@@ -143,6 +143,15 @@ def _applyBackendAttributeInt(self, attribute, taskConfig, portIndex, server, di
             else:
                 logging.warn("Weight %s=%s for task %s is not numeric ", attribKey, attribValue, taskConfig.get('id'))
 
+    def _applyBackendConnectionTimeout(self, attribute, taskConfig, portIndex, server):
```

Comment:
Would suggest removing this function again, and renaming _applyBackendAttributeInt() to just _applyAttributeInt() and then using that to extract and set the timeouts on the service object. Also change the function like

```
                setattr(server, attribute.replace('.', ''), int(attribValue) / divisor)
```